### PR TITLE
Feat: Add swahili language translation

### PR DESCRIPTION
**Pull request by letelete**
Fixes #Issue_Number

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by luckyman20 — COMMENTED** [review]
@letelete I see some of the strings aren't translated, any specific reason for that.

**Comment by letelete** [comment]
I've removed strings which, imho, shouldn't be translated like app name and location. Am I wrong with this?

**Comment by luckyman20** [comment]
App name and location are fine but strings like Password, filter etc. should be translated.

**Comment by letelete** [comment]
Oh, yeah, looks like I missed a couple of these. I'm going to translate them now.

**Comment by letelete** [comment]
Updated! Should be fine already.

File: app/src/main/res/values-sw/strings.xml [hunk 1]
```diff
@@ -1,272 +1,418 @@
...
+    <string name="logout">Ondoka kwenye</string>
+    <string name="share_msg">Pakua programu ya Self Service hapa: https://play.google.com/store/apps/details?id=</string>
+    <string name="choose">Chagua programu</string>
+    <string name="not_available">THE</string>
+    <string name="upload_qr_code">Weka QR</string>
+    <string name="select_region_qr">Chagua kanda na msimbo wa QR</string>
 
-    <!--Customer care details-->
-    <string name="need_help">need help? Contact your support line</string>
-    <string name="help_line_number">8000000000</string>
```
Comment (luckyman20) [inline app/src/main/res/values-sw/strings.xml]: Don't remove this string, keep it as it is.

File: app/src/main/res/values-sw/strings.xml [hunk 2]
```diff
@@ -1,272 +1,418 @@
...
 
-    <string name="account_short">A/C</string>
-    <string name="loan_product">Loan Product</string>
-    <string name="inactive">Inactive</string>
-    <string name="active_uc">ACTIVE</string>
-    <string name="inactive_uc">INACTIVE</string>
 
-    <!--Material Dialog-->
+    <string name="dialog_logout">Una uhakika unataka kuingia nje?</string>
     <string name="dialog_action_ok">OK</string>
```
Comment (luckyman20) [inline app/src/main/res/values-sw/strings.xml]: Translate this too

File: app/src/main/res/values-sw/strings.xml (hunk 1 above)
Comment (letelete) [reply app/src/main/res/values-sw/strings.xml]: Okay

**Comment by letelete** [comment]
Is there anything else I should fix?

**Comment by luckyman20** [comment]
@letelete Travis is failing, fix that.

**Comment by letelete** [comment]
It's passing locally. 

Travis logs: `Error: Invalid --abi armeabi-v7a for the selected target.` I'll try to recommit it because it's been passing earlier and I just changed the string.

image

**Review by luckyman20 — APPROVED** [review]

**Pull request merged**